src/locales/fr.js

```diff
@@ -49,7 +49,7 @@ export default {
     "corpusChristi": "Saint-Sacrement",
     "divineMercySunday": "Dimanche de la Divine Miséricorde",
     "easter": "Dimanche de Pâques",
-    "epiphany": "Dimanche des Rameaux et de la Passion",
+    "epiphany": "L'Épiphanie du Seigneur",
```

Comment:
@emagnier, as I have already replaced all `'` with `’`, could you replace it in here too please? :smiley:

`L’Épiphanie du Seigneur`

Reply:
Done.